packages/cdktf/lib/backends/remote-backend.ts

```diff
@@ -18,6 +18,17 @@ export class RemoteBackend extends TerraformBackend {
     return keysToSnakeCase({ ...this.props });
   }
 
+  public toMetadata() {
+    const cloud = [
+      undefined,
+      "app.terraform.io",
+      "app.terraform.io:80",
```

Comment:
Is :443 invalid?

Reply:
Good catch 😁
And yeah, with all those additions a substring might be good enough.
It would catch also `app.terraform.io.ansgar.dev` or [test.ansgar.dev/app.terraform.io](https://test.ansgar.dev/app.terraform.io) though 😁 But I agree that this will be "too rare to care" 😄